web/modules/custom/cybersecurity_common/cybersecurity_common.module

```diff
@@ -32,3 +34,58 @@ function cybersecurity_common_js_settings_alter(array &$settings) {
     }
   endif;
 }
+
+/**
+ * Generates a string representation for the given byte count.
+ *
+ * @param int $size
+ *   A size in bytes.
+ * @param string $langcode
+ *   Optional language code to translate to a language other than what is used
+ *   to display the page.
+ *
+ * @return \Drupal\Core\StringTranslation\TranslatableMarkup
+ *   A translated string representation of the size.
+ */
+function cyber_format_size($size, $langcode = NULL) {
```

Comment:
I would just like to see a bit more of an explanation here i.e. explain which core function you have copied and tweaked, and why you have done it.